classes/Language.php

```diff
@@ -1189,15 +1189,18 @@ public static function countActiveLanguages($id_shop = null)
         return static::$countActiveLanguages[$id_shop];
     }
 
-    public static function downloadAndInstallLanguagePack($iso, $version = _PS_VERSION_, $params = null, $install = true)
+    /**
+     * @param mixed $version Unused parameter @deprecated since 8.0.0 and will be removed in next major version.
+     */
+    public static function downloadAndInstallLanguagePack($iso, $version = null, $params = null, $install = true)
```

Comment:
Deprecate the parameter version (if $version != null)

https://devdocs.prestashop.com/1.7/development/coding-standards/#deprecations